.github/workflows/bridge-ui-e2e-tests.yml

```diff
@@ -6,11 +6,13 @@ on:
       - main
     paths:
       - 'bridge-ui/**'
+      - '!**.md'
   push:
     branches:
       - main
     paths:
       - 'bridge-ui/**'
+      - '!**.md'
```

Comment:
I think it should be `!bridge-ui/**/*.md`. WDYT ?

Reply:
fixed